src/coreclr/jit/lsrabuild.cpp

```diff
@@ -3035,7 +3035,7 @@ void LinearScan::UpdatePreferencesOfDyingLocal(Interval* interval)
 // Notes:
 //    The node must not be contained, and must have been processed by buildRefPositionsForNode().
 //
-RefPosition* LinearScan::BuildUse(GenTree* operand, regMaskTP candidates, int multiRegIdx)
+RefPosition* LinearScan::BuildUse(GenTree* operand, regMaskTP candidates, int multiRegIdx, bool needsConsecutive)
```

Comment:
Put `needsConsecutive` under `ARM64_ARG()`?

Add `needsConsecutive` to function header comment